src/NumberButton.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function NumberButton({ onClick, children }) {
+  return <button type="button" onClick={onClick}>{children}</button>;
```

Comment:
```suggestion
  return (
    <button
      type="button"
      onClick={onClick}
    >
        {children}
     </button>
   );
```
코드를 정리하는 기준이 있으신가요?
저는 `return ();` 형태로 기본 형태를 잡아주고, 속성이 2개 이상일 때는 줄바꿈을 해서 코드 가독성을 높이려고 하는 편이에요.
어떤게 더 보기 좋은지 비교해보고 자신만의 코드 정리 기준을 잡으시면 좋아요.

Reply:
코멘트를 보고 코드를 짜면서 이 코드를 읽는 사람에게 어떻게 코드가 읽힐지를 생각하지 않았다는 걸 깨달았습니다.
다음 부터는 읽는 사람에 코드가 어떻게 읽힐 지 고려하면서 코드를 짜려고 합니다!

코멘트 해주신 부분도 수정하겠습니다!